content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+    the committee before, or has not served on the committee for 4 or
+    more terms. Interviewers should be chosen by the current committee.
+    If an interviewer has a conflict of interest with the interviewee,
+    an alternate interviewer will be chosen. In any case, if no
+    interviewer is explicitly chosen, the current Chair shall conduct
+    interviews. The interviews are intended to welcome the new member to
+    the committee, give them background on the committee's activities,
+    and help them find a role for the next term. The current committee
+    will ultimately select the comoposition of the committee for next
+    term.
+
+3.  Existing committee members can nominate themselves for positions on
+    the next term's committee given the following conditions:
+
+    -   No member may serve more than three consecutive terms;
+
+    -   No member may serve as Chair for more than one term
+        consecutively;
+
+    -   No member intending to serve as Chair or Secretary may be on
+        co-op;
+
+    -   Members wishing to nominate themselves for chair should be a
+        previous WiCS committee member. If no such volunteer exists, the
+        position opens up to new members;
+
+    -   Excluding that of Chair, no positions may be chosen until the
+        first official meeting of the new term's committee;
+
+    -   Existing committee members may compose no more than 75% of the
+        new committee, if more existing members are nominated than 75%
+        can accomodate, each returning member will submit a new
+        application to be reviewed by the serving Director of the WiCS
+        Committee.
+
+4.  If there are any objections to an applicant, this procedure should
+    be followed:
+
+    1.  The objection will be presented to the committee, with
+        sufficient evidence;
+
+    2.  The person being objected to may present a defense;
+
+    3.  The current committee will then take a vote, if a majority of
+        members agree with the objection, the objected person will be
+        denied a position on the committee.
+
+    If the objection is a sensitive concern, it can be brought up to the
+    Chair or Secretary, where they will decide whether to bring the case
+    to the committee and how to proceed. Otherwise, the objected person
+    will be automatically accepted for the next term.
+
+5.  Once the committee has been chosen, the Director of the WiCS
+    Committee must ratify the new committeee.
+
+## Standing Committees ##
+
+-   The Systems Committee (wics-sys) shall:
+
+    1.  Be chaired by the systems administrator;
+
+    2.  Operate all equipment in the possession of the committee;
+
+    3.  Maintain and upgrade the software on the equipment used by the
+        committee.
+
+-   Other Committees\
+    The undergraduate committee can create and task standing committees
+    as necessary. Such standing committees can be dissolved by the
```

Comment:
Probably should note that they autodissolve when they fulfill their objective (typical for standing committees) and explicitly note that they can persist between terms, ala "standing".